README.md

```diff
@@ -1,5 +1,15 @@
+[![Stars](https://img.shields.io/github/stars/laminlabs/bionty?logo=GitHub&color=yellow)](https://github.com/laminlabs/bionty)
+[![pypi](https://img.shields.io/pypi/v/bionty?color=blue&label=pypi%20package)](https://pypi.org/project/bionty)
+
 # bionty: Basic biological entities
 
 bionty is documented & tested within lamindb: github.com/laminlabs/lamindb.
 
+- Access public ontologies of biological entities
+- Manage synonyms, abbreviations, and hierarchical relationships
```

Comment:
"manage hierarchical relationships" could be something along the lines of "manage registries and ontologies with one framework" 🤔 -- or whatever wording conveys to people that they have "both registries and ontologies in the same thing".